crates/bevy_ecs/src/schedule/executor/mod.rs

```diff
@@ -118,14 +126,119 @@ impl SystemSchedule {
 /// - This function (currently) does nothing if it's called manually or wrapped inside a [`PipeSystem`](crate::system::PipeSystem).
 /// - Modifying a [`Schedule`](super::Schedule) may change the order buffers are applied.
 #[doc(alias = "apply_system_buffers")]
-#[allow(unused_variables)]
-pub fn apply_deferred(world: &mut World) {}
+#[allow(unused_variables, non_upper_case_globals)]
```

Comment:
I think this change to apply_deferred should be it's own pr as it's really unrelated to the exclusive system trait change. It's also less controversial than the rest of this pr, so it could be merged quickly.

Reply:
Done: https://github.com/bevyengine/bevy/pull/16642